scripts/lib/mkcloud-libvirt.sh

```diff
@@ -112,8 +112,10 @@ done
 iptables_unique_rule PREROUTING -t nat -p tcp --dport 6080 \\
     -j DNAT --to-destination $net_public.2
 
-iptables_unique_rule FORWARD -d $net_admin.0/24 -j ACCEPT
-iptables_unique_rule FORWARD -d $net_public.0/24 -j ACCEPT
+for x in D I ; do
```

Comment:
OK thanks.  This extra information is very useful and I would prefer it to be in the commit message too, but I'll remove my -1 and leave you to decide whether you have the energy to do one more final force-push ;-)